contents/docs/user-guides/experimentation.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: Experimentation
+sidebar: Docs
+showTitle: true
+---
+
+<FeatureAvailability availablePlans={['standard', 'enterpriseCloud', 'startup', 'scale', 'enterprise']} />
+
+Experimentation helps you test product changes to your app and find the optimal changes. You can test hypotheses about which product changes will lead to optimal results. You may have heard it called A/B testing before. Examples:
+
+-   You want to test which copy or visual design has the highest conversion.
+-   You want to test which flow leads to the highest number of purchases.
+-   Another use case here?
```

Comment:
You want to test if certain deals & promotions do better with users in a specific country.

You want to test if showing your website copy in German does better than English in Austria.